pkg/cloud/openstack/services/networking/loadbalancer.go

```diff
@@ -274,14 +274,18 @@ func (s *Service) DeleteLoadBalancerMember(clusterName string, machine *clusterv
 
 	lbID := clusterProviderStatus.Network.APIServerLoadBalancer.ID
 
-	for _, port := range []int{22, 6443} {
+	for _, port := range []int{6443} {
```

Comment:
When we create the listener and pool, we are doing like:
```Go
portList := []int{clusterProviderSpec.APIServerLoadBalancerPort}
portList = append(portList, clusterProviderSpec.APIServerLoadBalancerAdditionalPorts...)
for _, port := range portList {
        lbPortObjectsName := fmt.Sprintf("%s-%d", loadBalancerName, port)
```
Please refer to https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/pkg/cloud/openstack/services/networking/loadbalancer.go#L84-L86

Reply:
Oh sorry, I though I already fixed that somewhere.

So the solution is to remove the same list of ports that have been added before